tests/test_sockets.py

```diff
@@ -971,27 +970,6 @@ async def test_tcp_listener_same_port(
 
             assert len(multi.listeners) == expected_listeners
 
-    @skip_ipv6_mark
-    async def test_tcp_listener_dualstack_disabled(
-        self, monkeypatch: MonkeyPatch
-    ) -> None:
-        monkeypatch.setattr(socket, "has_dualstack_ipv6", lambda: False)
-
-        async with await create_tcp_listener(
-            local_host=None, family=AddressFamily.AF_UNSPEC
-        ) as multi:
-            families = {
-                listener.extra(SocketAttribute.family) for listener in multi.listeners
-            }
-            assert families == {socket.AF_INET, socket.AF_INET6}
-            assert len(multi.listeners) == 2
-
-            ports = {
-                listener.extra(SocketAttribute.local_port)
-                for listener in multi.listeners
-            }
-            assert len(ports) == 1
-
```

Comment:
You said in the issue that you left this test intact, but clearly you didn't? What's going on?